test/images/scripts/openshift.setup

```diff
@@ -62,6 +62,8 @@ cat > /openshift-prep <<EOF
 #!/bin/sh -ex
 if /usr/sbin/ip addr | /usr/bin/grep -i "52:54:00:9e:00:f1"; then
     /usr/bin/hostnamectl set-hostname f1.cockpit.lan
+    # work around https://bugzilla.redhat.com/show_bug.cgi?id=1401561
```

Comment:
in other places we use the convention
`# HACK work around  rpcbind not starting https://bugzilla.redhat.com/show_bug.cgi?id=1401561`